helm/promtail/templates/cilium-networkpolicy.yaml

```diff
@@ -34,5 +34,14 @@ spec:
         rules:
           dns:
           - matchPattern: '*'
+    - toEndpoints:
+      - matchLabels:
+          app.kubernetes.io/name: loki
+      toPorts:
+      - ports:
+        - port: "80"
+          protocol: ANY
+        - port: "9095"
+          protocol: ANY
```

Comment:
Upstream chart supports nodeport service type for loki. 
https://github.com/grafana/loki/blob/main/production/helm/loki/values.yaml#L1382

Is that something we need to consider here or since this is a GS app we unlikely will ever use nodeports in our clusters.

Reply:
I think it can be ignored for now and maybe addressed in the future if it becomes a problem. It's always possible to add an extra CiliumNetworkPolicy if needed I think.